optimum/intel/openvino/quantization.py

```diff
@@ -104,9 +106,10 @@ def quantize(
         save_directory: Union[str, Path],
         quantization_config: OVConfig = None,
         file_name: Optional[str] = None,
-        batch_size: int = 8,
+        batch_size: int = 1,
```

Comment:
Yes, mostly for safety reasons, especially for LLMs. It should not hurt performance since it is just a matter of 300 inferences by default and batch size does help here that much.